samples/net/sockets/echo_client/sample.yaml

```diff
@@ -78,10 +78,20 @@ tests:
       - openthread
     platform_allow:
       - frdm_kw41z
-      - frdm_mcxw71
       - nrf52840dk/nrf52840
       - tlsr9518adk80d
     filter: CONFIG_FULL_LIBC_SUPPORTED and not CONFIG_NATIVE_LIBC
+  sample.net.sockets.echo_client.mcxw_openthread:
+    extra_args:
+      - EXTRA_CONF_FILE=./boards/frdm_mcxw71.conf
+      - DTC_OVERLAY_FILE=./boards/frdm_mcxw71.overlay
```

Comment:
That shouldn't be really needed, board overlays should be applied automatically by the build system.

Reply:
Removed the `DTC_OVERLAY_FILE`. Thank you